variables.tf

```diff
@@ -292,9 +292,11 @@ variable "addons" {
     static-route              = optional(string)
     cluster-autoscaler        = optional(string)
     vpc-block-csi-driver      = optional(string)
+    ibm-storage-operator      = optional(string)
   })
   description = "Map of OCP cluster add-on versions to install (NOTE: The 'vpc-block-csi-driver' add-on is installed by default for VPC clusters, however you can explicitly specify it here if you wish to choose a later version than the default one). For full list of all supported add-ons and versions, see https://cloud.ibm.com/docs/containers?topic=containers-supported-cluster-addon-versions"
```

Comment:
I guess this description should also mention that `ibm-storage-operator` is installed by default in OCP 4.15 and later